t/25-spvm.t

```diff
@@ -36,4 +38,19 @@ subtest 'SSH credentials in spvm' => sub {
     is($spvm->run_cmd('true', $expected_credentials->{hostname}, $expected_credentials->{password}), 0, "Test specific credentials");
 };
 
+subtest 'PowerVM power actions' => sub {
+    my $mock_spvm = Test::MockModule->new('backend::spvm');
+    $mock_spvm->mock('run_cmd', sub {
+            my ($self, $cmd) = @_;
+            return $cmd;
+    });
+
+    my $spvm    = backend::spvm->new();
+    my $lpar_id = 3;
+    set_var(NOVALINK_LPAR_ID => $lpar_id);
+
+    is($spvm->power({action => 'on'}), "pvmctl lpar power-on -i id=${lpar_id} --bootmode norm", "Test power on");
+
+    throws_ok { $spvm->power({action => 'reboot'}) } qr/Unknown power action reboot/, 'Unknown power action';
```

Comment:
If you don't mind then I suggest to remove the blank lines in between, but only if you don't mind :)